region/maridia/outer/Mt. Everest.json

```diff
@@ -2123,6 +2016,41 @@
       ],
       "gModeRegainMobility": {}
     },
+    {
+      "link": [6, 7],
```

Comment:
I think these two [6, 7] strats could be simplified to a [6, 2] strat that just falls down, since you can Gravity jump from 2 to 7.